FacesofLondon/app/src/main/java/com/novoda/wallpaper/droidcon/TimeOfDayCalculator.java

```diff
@@ -1,12 +1,18 @@
 package com.novoda.wallpaper.droidcon;
 
-import java.util.Calendar;
-
 class TimeOfDayCalculator {
```

Comment:
I think you are right. Since `TimeOfDay.forHourOfDay(hourOfDay)` was moved into the same class, might as well move this there too